adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/VersionData.kt

```diff
@@ -45,12 +49,14 @@ class VersionData : Comparable<VersionData> {
         var semver = major.toString() + "." + minor + "." + security
 
         if (pre?.isNotEmpty() == true) {
-            semver += "-" + pre
+            semver += "-$pre"
         }
 
         var metadata = listOf<String>()
+        val buildOffset = if (patch != null && patch != 0) patch * 100 else 0
```

Comment:
comment as to why 100

Reply:
maybe even extract to a named const

Reply:
this matches the change made in the build repo that produced a build of 101